### PR TITLE
Orderpage

### PR DESCRIPTION
**Pull request by ipsitapatraa**
Hi Sourav,
can you please chek pr?
++ added order page
thanks 
ip

### CODE REVIEW

File: src/OrderPage.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+public class OrderPage {
+
+	public static void main(String[] args) {
+		System.out.println("ordepage..");
+		System.out.println("ordepage commite 2..");
```
Comment (SouravLab) [inline src/OrderPage.java:6]: where is your method?
where is your logic?

**Review by SouravLab — CHANGES_REQUESTED** [review]
hi ip
please check my review comment and make he changes
thanks 
sourav

File: src/OrderPage.java (hunk 1 above)
Comment (ipsitapatraa) [reply src/OrderPage.java:6]: done ..logic added.rocket is ready for launch

Comment (ipsitapatraa) [reply src/OrderPage.java:6]: added

Comment (SouravLab) [reply src/OrderPage.java:6]: thanks ..i wiil do tthe merge,,,..

**Review by SouravLab — APPROVED** [review]
code is working fine

**Comment by SouravLab** [comment]
code reviewed and approved by sourav
merging now..

**Pull request merged**